server/service/appconfig.go

```diff
@@ -672,6 +694,20 @@ func (svc *Service) validateMDM(
 		invalid.Append("macos_updates", err.Error())
 	}
 
+	// WindowsUpdates
+	updatingWindowsUpdates := !mdm.WindowsUpdates.Equal(oldMdm.WindowsUpdates)
+	if updatingWindowsUpdates {
+		// TODO: Should we validate MDM configured on here too?
```

Comment:
It would nice to have a Product/QA-focued initiative to define all the expected validations in a central place. That way we could then estimate a ticket for us to put that into practice with robust tests. And then going forward, it can be incorporated into the design process to specify new/changed validations.

Reply:
Agreed. For now I added the same TODO you added elsewhere where it was lacking, so if we want to add that validation we can address them all in one PR.